### ISSUE TITLE
Enhancement: ctrl-alt from room connectors?

### ISSUE THREAD

**Issue by andrewschultz** [JasonLautzenheiser/trizbort#94]
It would be nice to be able to ctrl-arrow from a connector to either room it connects.

Test case.
1) create new file
2) ctrl-left
3) click on the connector.

Note ctrl-left or ctrl-right doesn't move you to a new room, which would be convenient if, say, we wanted to edit a connector right after making a room.

**Comment by andrewschultz** [comment]
I have an initial fix for this. I found a test case where it still doesn't work, but it's pretty minor (e.g. if you create 2 rooms and connect one's north port with one's east port, ctrl-up sends you to the east room and ctrl-left sends you down. I think I'm choosing the wrong connections.) Would this be ok to submit now and fix the known bug later?

**Comment by andrewschultz** [comment]
OK, I think I found this. But whatever review you have would be welcome, because I suspect I duplicated some code. I'm curious about if we can/should pull the GetAutomapOpposite function in CompassPoint.cs to Canvas.cs. (I didn't run the simple cut/paste to check.) Or assignprimarydirection in CodeExporter.cs to Canvas.cs. (My knowledge/understanding of C# structures is still very bad. If there's a good resource to read, let me know!)
